build.sbt

```diff
@@ -15,25 +14,23 @@ def isScala210x(scalaVersion: String) = scalaBinaryVersion(scalaVersion) == "2.1
 def sequentialExecution: Boolean =
   Option(System.getProperty("sequentialExecution")).map(_.toBoolean).getOrElse(false)
 
-val algebirdVersion = "0.12.0"
-val bijectionVersion = "0.9.1"
-val chillVersion = "0.8.3"
-val commonsHttpClientVersion = "3.1"
+val algebirdVersion = "0.13.0"
+val bijectionVersion = "0.9.5"
+val chillVersion = "0.8.4"
 val commonsLangVersion = "2.6"
-val finagleVersion = "6.35.0"
 val hadoopVersion = "1.2.1"
 val junitVersion = "4.11"
 val log4jVersion = "1.2.16"
 val novocodeJunitVersion = "0.10"
 val scalaCheckVersion = "1.13.4"
 val scalatestVersion = "3.0.1"
-val scaldingVersion = "0.16.1-RC3"
+val scaldingVersion = "0.17.1"
```

Comment:
`0.17.0` breaks SB tests so we can't roll back. I'm going to publish `0.17.2` today.